src/deckgl-layers/layer-utils/shader-utils.js

```diff
@@ -18,22 +18,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import {console as Console} from 'global/window';
+
 /*
  * Amendment to default layer vertex shader
  * @param {string} vs
- * @param {bool} opt.highlightPicked - whether to highlight picked element
+ * @param {string} type
+ * @param {string} originalText
+ * @param {string} testToReplace
+ * @return {string} output shader
  *
  */
-export function getCellLayerVertex(vs, {highlightPicked}) {
-  let output = vs;
-
-  if (highlightPicked) {
-    output = output.replace(
-      'vec3 lightWeightedColor = lightWeight * instanceColors.rgb;',
+export function editShader(vs, type, originalText, testToReplace) {
```

Comment:
With v6 you can use shader injection:
https://github.com/uber/deck.gl/blob/master/test/apps/data-filter/src/scatterplot-layer.js#L15

See [list of injection points](https://github.com/uber/luma.gl/blob/master/modules/core/src/shadertools/src/lib/inject-shader.js#L22)